scripts/run-python-tests.sh

```diff
@@ -10,9 +10,4 @@ set -eu
 scriptdir=$(readlink -f "$(dirname "$0")")
 cd "$scriptdir/.."
 
-. "$scriptdir/util.sh"
-pythoncmd=`getPythonCmd`
-
-# Linux filesystems are case-sensitive, so we need to tell the python unit test
-# scanner to look for upper-case files as well:
-$pythoncmd -m unittest discover --verbose --locals --failfast -s source/Mlos.Python -p "[Tt]est*.py" $*
+pytest -svx source/Mlos.Python
```

Comment:
```suggestion
pytest -svx source/Mlos.Python $*
```

If we keep the trailing `$*` we can continue to pass `-k someTest` to through the script to be able to limit the set of tests it runs.

Given the command is much shorter/easier now, this is probably less of a concern as it was before though.